test/sql/function/timestamp/test_date_part.test

```diff
@@ -739,6 +739,16 @@ NULL	NULL
 2022-01-01 00:00:41+00	1640995241000
 infinity	NULL
 
+query I
```

Comment:
This should throw an error I think, but its not part of this issue.

Reply:
Made it throw an error 👍